### DISCUSSION TITLE
How to train cyclegan under Win10?

### DISCUSSION THREAD

**Discussion by yuninn** [open-mmlab/mmgeneration#452] [How-to] [answered]
.sh file cannot be run directly under Win10, so how can I train my model directly through train.py in Win10?

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by plyfager** [comment, marked as answer]
@yuninn  Hi, you may try powdershell.

**Comment by yuninn** [comment]
> @yuninn Hi, you may try powdershell.

Thank you for your reply!
After：
bash tools/dist_train.sh configs/cyclegan/cyclegan_lsgan_resnet_in_bktrans_b1x1_250.py 1 --work-dir work_dirs/exp/1/

I have a new problem：

subprocess.CalledProcessError: Command '['E:\Anaconda3\envs\open-mmlab\python.exe', '-u', 'tools/train.py', '--local_rank=0', 'configs/cyclegan/cyclegan_lsgan_resnet_in_bktrans_b1x1_250.py', '--launcher', 'pytorch' '--work-dir work_dirs/exp/1/']' returned non-zero exit status 1.
